Pre-Processing:_Change_format_#26.ipynb

```diff
@@ -0,0 +1,74 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 48,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def change_format(Image_path, picture_format):\n",
+    "    \"\"\"\n",
+    "    Function that changes the image format. It takes the image path and the desired format as input. It can only switch .png and .jpeg files, others will raise an error.\n",
+    "    \"\"\"\n",
+    "    # example call: change_format(\"/Users/XX/Desktop/Screenshot 2023-06-23 at 15.18.03.jpeg\", \".png\")\n",
+    "\n",
+    "    # installs the PIL package if not already there. This adds image processing capabilities (see also https://pypi.org/project/Pillow/, doc: https://pillow.readthedocs.io/en/stable/reference/Image.html)\n",
```

Comment:
Leaving that in the function or maybe including it in our requirements and removing it here?